Bottom: c6034d169052a932cd9ef99c15e65a5b89cc71f8
Top:    0affd7ec32b2b01f25de642492a663fdfbf17fd5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-04-05 22:30:42 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index b20cd20..a4797b8 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -312,13 +312,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
